### PR TITLE
backoff 1.4.3 (PyPi)

### PR DESCRIPTION
**Pull request by gatesn**

### CODE REVIEW

File: recipes/backoff/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    - python
+
+test:
+  imports:
+    - backoff
+
+about:
+  home: https://github.com/litl/backoff
+  license: MIT
+  license_family: MIT
```
Comment (synapticarbors) [inline recipes/backoff/meta.yaml]: Please add the `license_file` as described here:

https://github.com/litl/backoff

and also possibly add the license file to the upstream `MANIFEST.in` so it's packaged with releases of `backoff` in the future (the recipe can be updated at that point to use the one that comes with the source).

File: recipes/backoff/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    This module provides function decorators which can be used to wrap a function such that it will be retried until
+    some condition is met. It is meant to be of use when accessing unreliable resources with the potential for
+    intermittent failures i.e. network resources and external APIs. Somewhat more generally, it may also be of use for
+    dynamically polling resources for externally generated content.
+  dev_url: https://github.com/litl/backoff
+
+extra:
+  recipe-maintainers:
+    - gatesn
+    - bgreen-litl
```
Comment (synapticarbors) [inline recipes/backoff/meta.yaml]: @bgreen-litl are you ok with being added as a co-maintainer of the `backoff` package on conda-forge?

File: recipes/backoff/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
+{% set name = "backoff" %}
+{% set version = "1.3.1" %}
```
Comment (synapticarbors) [inline recipes/backoff/meta.yaml]: The latest release is `1.4.2`. Any reason to be packaging an older version?

File: recipes/backoff/meta.yaml (hunk 2 above)
Comment (gatesn) [inline recipes/backoff/meta.yaml]: I have spoken to @bgreen-litl offline and confirmed this is ok.

Comment (bgreen-litl) [inline recipes/backoff/meta.yaml]: Yes.  Thank you.

File: recipes/backoff/meta.yaml (hunk 1 above)
Comment (bgreen-litl) [inline recipes/backoff/meta.yaml]: I added the LICENSE to the manifest, and just released backoff 1.4.3 which now includes the license file in the sdist tarball.

**Comment by bgreen-litl** [comment]
@gatesn if you set the version to backoff 1.4.3 now the LICENSE file should automatically be included.  Do we know what the failing checks are about?

**Comment by gatesn** [comment]
@synapticarbors have bumped to 1.4.3, the version that includes the LICENSE file in the manifest.

**Pull request merged**